### PR TITLE
Cleanup test code; added test script to package.json

### PR DESCRIPTION
**Pull request by yjcxy12**
I've noticed test folder was added yet there's no script to run the tests. So I tried to lint the code and add some scripts.

Although one test is failing, and as I did not write these tests originally, I did not change the failing test.

### CODE REVIEW

File: test/server/utils/user-stats-test.js [hunk 1]
```diff
@@ -1,132 +1,126 @@
...
-    ]), 3, "should return 3 when today and for two days before user was activity");
+    ]), 3, 'should return 3 when today and for two days before user was activity');
 
   t.equal(calcCurrentStreak([
       moment(moment().subtract(37, 'hours')).valueOf(),
       moment(moment().subtract(1, 'hours')).valueOf()
-    ]), 1, "should return 1 when between todays challenge completion and yesterdays there is a 1.5 day (36 hours) long break");  
+    ]), 1, 'should return 1 when between todays challenge completion and yesterdays there is a 1.5 day (36 hours) long break');  
 
   t.equal(calcCurrentStreak([
```
Comment (yjcxy12) [inline test/server/utils/user-stats-test.js]: This test was failing on my machine with actual: 3, expected: 2

**Comment by bugron** [comment]
@FreeCodeCamp/issue-moderators can you, please, check, why is this PR failing the CI build and QA this PR?

**Comment by SaintPeter** [comment]
@bugron Looks like it's passing now.

**Comment by bugron** [comment]
@SaintPeter you're a magician)) Can you QA, please?

**Comment by SaintPeter** [comment]
I am unable to run the tests on Windows for reasons that remain opaque but can probably summed up as "It's Windows".

Someone with a real OS will need to QA this, sorry.

**Comment by bugron** [comment]
I'm on Windows too.

**Comment by bugron** [comment]
Closing this PR as stale. @yjcxy12 feel free to reopen or create a new PR, thanks.

**Pull request closed without merging**